### ISSUE TITLE
Will Unreal Engine 5.1 and Ubuntu 20.04 be supported in the future?

### ISSUE THREAD

**Issue by YuminosukeSato** [Akiya-Research-Institute/3dGaussiansPlugin-Demo#5]
Will this plugin be compatible with systems other than Windows and unreal engine 5.1 and 5.2 in the future? For example, does it support Unreal Engine 5.1 and Ubuntu 20.04?

**Comment by KenjiAsaba** [comment]
> Unreal Engine 5.1

Yes, I plan to add support for UE5.1 and 5.2 in the next week or so.
We are using UE5.1 for development.

> Ubuntu 20.04

Possibly yes, but I cannot gurantee.
See [FAQ](https://akiya-research-institute.github.io/3dGaussiansPlugin-Manual/en/faq/#how-can-i-make-packages-for-platforms-other-than-windows).
